cmd/gateway/azure/gateway-azure.go

```diff
@@ -125,6 +131,19 @@ func azureGatewayMain(ctx *cli.Context) {
 	minio.StartGateway(ctx, &Azure{host})
 }
 
+func getUploadChunkSizeFromEnv(envvar string, defaultValue int) int {
```

Comment:
Can you add a comment about the purpose of this function? @boazhuisman

Reply:
Done